test/NonSilo.Tests/SiloHostBuilderTests.cs

```diff
@@ -168,6 +168,8 @@ private static void RemoveConfigValidators(IServiceCollection services)
         {
             var validators = services.Where(descriptor => descriptor.ServiceType == typeof(IConfigurationValidator)).ToList();
             foreach (var validator in validators) services.Remove(validator);
+            // Configure endpoints because validator is set just before Build()
+            services.Configure<EndpointOptions>(options => options.AdvertisedIPAddress = IPAddress.Loopback);
```

Comment:
This is a slightly confusing place to put this code, since the method is called `RemoveValidators`.

You could rename the method or put the code somewhere else